OutOfSchool/OutOfSchool.BusinessLogic/Services/Memento/JSONConverter/JSONCreationConverter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace OutOfSchool.BusinessLogic.Services.Memento.JSONConverter;
+
+public abstract class JsonCreationConverter<T> : JsonConverter
+{
+    public override bool CanWrite
```

Comment:
(Code style) I think it will be more readable if you make it
`public override bool CanWrite { get => false; }`

Reply:
O.k. I changed code - used an exspression body for this method and CanConvert().